com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -333,7 +333,7 @@ public static void NetworkHide(List<NetworkObject> networkObjects, ulong clientI
 
         private void OnDestroy()
         {
-            if (NetworkManager.Singleton != null && NetworkManager.Singleton.SpawnManager.SpawnedObjects.ContainsKey(NetworkObjectId))
+            if (NetworkManager.Singleton != null && NetworkManager.Singleton.SpawnManager != null && NetworkManager.Singleton.SpawnManager.SpawnedObjects.ContainsKey(NetworkObjectId))
```

Comment:
This fixes an issue where it is possible to be destroying after the NetworkSpawnManager no longer exists.

Reply:
@TwoTenPvP might also be interested while he's working on singleton removal tasks these days.